packages/wouter/src/memory-location.js

```diff
@@ -39,9 +39,15 @@ export const memoryLocation = ({
     navigate,
   ];
 
+  function reset() {
+    history.splice(0, history.length);
+    history.push(path);
```

Comment:
Shouldn’t reset trigger the update event and set the location? Imo, when called while the app is still mounted we should rerender everything to be consistent with the new location